### PR TITLE
Issues/147

### PR DESCRIPTION
**Pull request by kptdobe**

### CODE REVIEW

File: src/default.pre.js [hunk 1]
```diff
@@ -34,6 +37,32 @@ function collectMetadata(ctx) {
...
+ */
+function collectNav(ctx) {
+    const params = {
+        org: ctx.strandConfig.urls.content.owner,
+        repo: ctx.strandConfig.urls.content.name,
+        tree: ctx.strandConfig.urls.content.ref,
+        path: 'SUMMARY.md'
+    };
+
+    return md2json.main(params).then(info => {
```
Comment (trieloff) [inline src/default.pre.js:52]: @kptdobe I think this is the wrong way to do it – every template script should render only one single resource and never try to go back to the repository. 
You can handle this much better though an edge-side include (take a look at the nav.md example in `template2lamdba`, with better overall performance.

**Review by trieloff — COMMENTED** [review]
Should be done differently, but I'd merge it anyway.

File: src/default.pre.js (hunk 1 above)
Comment (trieloff) [reply src/default.pre.js:52]: I've filed adobe/project-helix#171 and #4 for this

Comment (kptdobe) [reply src/default.pre.js:52]: Yes, I agree. Step by step... for now, that's the only way to do it.

**Pull request merged**